integration-tests/Cargo.toml

```diff
@@ -66,7 +66,7 @@ performance_stats = [
   "near-network/performance_stats",
 ]
 expensive_tests = []
-test_features = ["nearcore/test_features"]
+test_features = ["nearcore/test_features", "near-chunks/test_features"]
```

Comment:
I'd rather

```toml
diff --git a/chain/client/Cargo.toml b/chain/client/Cargo.toml
index 68bb99de9..3a854466d 100644
--- a/chain/client/Cargo.toml
+++ b/chain/client/Cargo.toml
@@ -54,6 +54,7 @@ expensive_tests = []
 test_features = [
   "near-network/test_features",
   "near-chain/test_features",
+  "near-chunks/test_featurs",
 ]
 delay_detector = [
   "near-chain/delay_detector",
diff --git a/integration-tests/Cargo.toml b/integration-tests/Cargo.toml
index fce626a17..c69cb0ba8 100644
--- a/integration-tests/Cargo.toml
+++ b/integration-tests/Cargo.toml
@@ -66,7 +66,7 @@ performance_stats = [
   "near-network/performance_stats",
 ]
 expensive_tests = []
-test_features = ["nearcore/test_features", "near-chunks/test_features"]
+test_features = ["nearcore/test_features"]
 protocol_feature_alt_bn128 = [
     "near-primitives/protocol_feature_alt_bn128",
     "node-runtime/protocol_feature_alt_bn128",
```

we should try to make it impossible to compile only *some* crates with `test_features`.